docker-compose.arm64.yaml

```
@@ -0,0 +1,163 @@
+      - GF_USERS_ALLOW_SIGN_UP=false
+    healthcheck:
+      test: ['CMD', 'wget', '-qO-', 'http://localhost:3000/api/health']
+      interval: 10s
+      timeout: 5s
+      retries: 3
+      start_period: 10s
+    depends_on:
+      - prometheus
+    networks:
+      - nestjs-network
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)